server/src/main/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilder.java

```diff
@@ -110,7 +110,7 @@ public static FieldMaskingSpanQueryBuilder fromXContent(XContentParser parser) t
                         throw new ParsingException(parser.getTokenLocation(), "[field_masking_span] query must be of type span query");
```

Comment:
Can we replace the name in these error messages as well?

Reply:
Sure, i'll fix those messages in my next commit